packages/@react-aria/overlays/test/usePreventScroll.test.js

```diff
@@ -48,6 +48,22 @@ describe('usePreventScroll', function () {
     expect(document.documentElement).not.toHaveStyle('overflow: hidden');
   });
 
+  it('should work with nested/multiple modals regardless of unmount order', function () {
```

Comment:
This test is similar to the one above it, but with the unmount order flipped. Confirmed it fails with the old usePreventScroll code